services/analyzer/app.py

```
@@ -49,109 +180,178 @@
+            else:
+                logger.warning(
+                    f"ML models not found at {MODEL_PATH}. "
+                    "Running heuristic-only mode."
+                )
+        except Exception as exc:
+            logger.error(f"Failed to load ML models ({exc}). Falling back to heuristics.")
+    else:
+        logger.warning("joblib/numpy not installed — ML detection disabled.")
+
+    # Background tasks
     asyncio.create_task(process_event_queue())
```

Ruff (0.15.0)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)